src/components/title/show/title-show.js

```diff
@@ -218,15 +211,6 @@ class TitleShow extends Component {
           lastMenu={this.lastMenu}
           bodyContent={(
             <Fragment>
-              <TagsAccordion
```

Comment:
I guess the `<Fragment>` is no longer needed since the `<TagsAccordion />` was removed

Reply:
Thanks @MaksymDryha -- removed it!